lib/stacker/resolvers/global_var_resolver.rb

```diff
@@ -0,0 +1,17 @@
+require 'stacker/resolvers/resolver'
+
+module Stacker
+  module Resolvers
+
+    class GlobalVarResolver < Resolver
+
+      def resolve
+        global_defaults = region.defaults.fetch 'globalParameterFile', {}
+        global_vars = YAML.load_file(global_defaults)
```

Comment:
do we need to check if `global_defaults` is empty before calling `load_file`?

Reply:
if they're using this then it's assumed the global var file exists, kind of like how we handle files